commercetools-internal-processors/src/main/java/io/sphere/sdk/annotations/processors/generators/UpdateActionsGenerator.java

```diff
@@ -0,0 +1,107 @@
+package io.sphere.sdk.annotations.processors.generators;
+
+import com.squareup.javapoet.*;
+import io.sphere.sdk.annotations.processors.models.PropertyGenModel;
+import io.sphere.sdk.commands.UpdateActionImpl;
+import org.apache.commons.lang3.StringUtils;
+
+import javax.annotation.Generated;
+import javax.annotation.Nullable;
+import javax.lang.model.element.ExecutableElement;
+import javax.lang.model.element.Modifier;
+import javax.lang.model.element.TypeElement;
+import javax.lang.model.util.Elements;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class UpdateActionsGenerator extends AbstractMultipleGenerator {
+
+    public UpdateActionsGenerator(final Elements elements) {
+        super(elements);
+    }
+
+    @Override
+    protected String getPackageName(final TypeElement annotatedTypeElement) {
+        return super.getPackageName(annotatedTypeElement).concat(".commands.updateactions");
+    }
+
+    @Override
+    public List<TypeSpec> generateTypes(final TypeElement annotatedTypeElement) {
+        final List<ExecutableElement> propertyMethods = getPropertyMethodsSorted(annotatedTypeElement);
+        final List<TypeSpec> typeSpecList = propertyMethods.stream()
+                .filter(m -> isPrimitiveType(PropertyGenModel.of(m).getType()))
+                .map(propertyMethod -> {
+                    final PropertyGenModel property = PropertyGenModel.of(propertyMethod);
```

Comment:
How about extracting this lambda into a separate method? Then we could replace the lambda with a method handle to the extracted method. Let's discuss this separately.